DelfinAnomalyDetection/DelfinKafkaAIBasicPoC.md

```diff
@@ -0,0 +1,49 @@
+#Delfin-Kafka-AI PoC
+
+-------
+
+### Introduction
+This provides basic PoC to illustrate Delfin-Kafka-AI integration where user can do analytics on metrics published by delfin.
+The example connects metric published by Delfin on Kafka with basic anomaly detection system.
+Anomaly detection module checks for spikes(anomaly) in collected metrics.
+
+### Architecture
+
+````
+ ---------            ---------            ------------------
+| Delfin  | ------>  |  Kafka  |  ----->  | Anomaly Detection |
+ ---------            ---------            -------------------
+````
+
+- Delfin publish metric data on kafka (Default topic: delfin-kafka)
+- Anomaly detection module subscribe for the topic and store the metric data in dictionary
+- Anomaly detections runs on stored data in dictionary
+
+### Steps:
+- Setup Delfin and Kafka with help of [documentation](https://docs.sodafoundation.io/guides/developer-guides/delfin/)
```

Comment:
This link does not give setup info for delfin and kafka!

Reply:
Updated link with https://github.com/sodafoundation/delfin/tree/master/installer#test-the-running-delfin-setupprocess